src/DataViews/LiveFunctionsDataViewTest.cpp

```diff
@@ -7,6 +7,8 @@
 #include <absl/hash/hash.h>
 #include <absl/strings/str_format.h>
 #include <absl/time/time.h>
+#include <gmock/gmock-actions.h>
+#include <gmock/gmock-matchers.h>
```

Comment:
Nit: These includes were probably added by clangd, but it is wrong. In GMock, only `gmock/gmock.h` is a public header and should only be included.